CMakeLists.txt

```diff
@@ -159,7 +161,7 @@ if(SELF_ENABLE_DOUBLE_PRECISION)
 endif()
 
 if(SELF_ENABLE_GPU)
-    message("Enabling GPU Support")
+    message("-- Enabling GPU Support")
```

Comment:
Would you prefer `-- SELF Build System : Enabling GPU Support` to be consistent with line 149?

Reply:
Good catch here. Making an adjustment to reflect this

Reply:
Fixed in [3738412](https://github.com/FluidNumerics/SELF/pull/53/commits/37384125cd5dd1e4542576d1b53b8a362304e2a2)